src/spec.cr

```diff
@@ -100,6 +109,14 @@ OptionParser.parse do |opts|
       exit 1
     end
   end
+  opts.on("--order MODE", "run examples in random order by passing MODE as 'random' or to a specific seed by passing MODE as the seed value") do |mode|
+    if mode =~ /\A(?:default|random|\d{1,5})\Z/
```

Comment:
A Reged is not appropriate here. You should rather compare the string values directly, and else try to parse as an int.